x-pack/docs/en/security/auditing/event-types.asciidoc

```diff
@@ -144,13 +240,79 @@ ones):
 
 The events with `event.type` equal to `ip_filter` have one of the following `event.action`
 attribute values: `connection_granted` or `connection_denied`.
-These event types also have the following extra attributes (in addition to the common
+These events also have the following extra attributes (in addition to the common
 ones):
 
 `transport_profile`   ::    The transport profile the request targeted.
 `rule`                ::    The <<ip-filtering, IP filtering>> rule that denied
                             the request.
 
+==== Audit event attributes of the security_config_change event type
+
+The events with the `event.type` attribute equal to `security_config_change` have one of the following
+`event.action` attribute values: `put_user`, `change_password`, `put_role`, `put_role_mapping`,
+`change_enable_user`, `change_disable_user`, `put_privileges`, `create_apikey`, `delete_user`,
+`delete_role`, `delete_role_mapping`, `invalidate_apikeys`, or `delete_privileges`.
+These events also have *one* of the following extra attributes (in addition to the common
+ones). The attributes' value is a nested JSON object:
```

Comment:
```suggestion
ones). The attribute's value is a nested JSON object:
```

There is only a single attribute isn't there?

Reply:
Yes, only one of them.